### ISSUE TITLE
custom Objective with Akaike Information Criterion

### ISSUE THREAD

**Issue by melitong** [keras-team/keras-tuner#551]
Hello, I am new in Keras-tuner. I've read several post regarding custom objective, it stated that we have to override def run_trial. am I True?

Suppose I wanna make Akaike Information Criterion (AIC) as custom objective, which need number of parameter, number of observation, and log loss (binary cross entropy) from the model, how can I get the log loss value in run trial? and calculate the AIC in run_trial? 
I hope somebody can help me. Thank you very much.

@ben-arnao

**Comment by haifeng-jin** [comment]
duplicate in the discussion
https://github.com/keras-team/keras-tuner/discussions/533
